frontend/src/hooks/useLookupCodes.tsx

```diff
@@ -38,7 +50,9 @@ function useCodeLookups() {
     getOptionsByType: getOptionsByType,
```

Comment:
Why not be consistent in syntax?

Reply:
Not sure what you mean here.

Reply:
Maybe I'm missing something, but it just caught my eyes when scanning the code.  It appeared inconsistent.
```
{
    getOptionsByType: getOptionsByType,
    getCodeById: getCodeById,
    getByType,
    getPublicByType,
    filterByParent,
    lookupCodes,
  }
```
Why not
```
{
    getOptionsByType,
    getCodeById,
    getByType,
    getPublicByType,
    filterByParent,
    lookupCodes,
}
```
or
```
{
    getOptionsByType: getOptionsByType,
    getCodeById: getCodeById,
    getByType: getByType,
    getPublicByType: getPublicByType,
    filterByParent: filterByParent,
    lookupCodes: lookupCodes,
  }
```

Reply:
Oh I see. Probably just multiple people touching the code. I'll update for consistency.